docs/integrations/sources/aws-cloudtrail.md

```diff
@@ -0,0 +1,48 @@
+# AWS CloudTrail
+
+## Overview
+
+The AWS CloudTrail source supports both Full Refresh and Incremental syncs. You can choose if this connector will copy only the new or updated data, or all rows in the tables and columns you set up for replication, every time a sync is run.
+
+This Source Connector is based on a [Boto3 CloudTrail](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudtrail.html).
+
+### Output schema
+
+This Source is capable of syncing the following core Streams:
+
+* [Management Events](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudtrail.html#CloudTrail.Client.lookup_events)
+
+Insight events are not supported right now. Only Management events are available.
+
+### Data type mapping
+
+| Integration Type | Airbyte Type | Notes |
+| :--- | :--- | :--- |
+| `string` | `string` |  |
+| `number` | `integer` |  |
+| `array` | `array` |  |
+| `object` | `object` |  |
+
+### Features
+
+| Feature | Supported?\(Yes/No\) | Notes |
+| :--- | :--- | :--- |
+| Full Refresh Sync | Yes |  |
+| Incremental - Append Sync | Yes |  |
```

Comment:
Incremental-append is the old wording. Incremental is the correct new wording. You might need to update the docs template. 
```suggestion
| Incremental Sync | Yes |  |
```